bpf/include/bpf/ctx/xdp.h

```diff
@@ -136,7 +136,8 @@ __csum_replace_by_4(__sum16 *sum, __wsum from, __wsum to)
 }
 
 static __always_inline __maybe_unused int
-l3_csum_replace(struct xdp_md *ctx, __u64 off, const __u32 from, __u32 to,
+l3_csum_replace(const struct xdp_md *ctx, __u64 off, const __u32 from,
+		__u32 to,
 		__u32 flags)
```

Comment:
small nit: could go to prior line

Reply:
:thinking: Not sure what happened here. Coccinelle is supposed to be good at handling this sort of things.